### PR TITLE
Fixing admin_key rest example in the design

### PR DESCRIPTION
**Pull request by mgoelswirlds**
**Description**:
 The rest example in the dynamic address book displayed a wrong admin_key. This PR fixes that.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: docs/design/dynamic-addressbook.md [hunk 1]
```diff
@@ -102,7 +102,10 @@ Response:
 {
   "nodes": [
     {
-      "admin_key": "0x565fd764f0957fa170a676210c9bdbddf3bc9519702cf927fa6767a40463b96f",
+      "admin_key": {
+        "_type": "ProtobufEncoded",
+        "key": 10101
```
Comment (steven-sheehy) [inline docs/design/dynamic-addressbook.md:107]: This is a string hexadecimal field (no '0x' prefix). Probably we should show an ED25519 variant and not ProtobufEncoded.

File: docs/design/dynamic-addressbook.md [hunk 2]
```diff
@@ -102,7 +102,10 @@ Response:
 {
   "nodes": [
     {
-      "admin_key": "0x565fd764f0957fa170a676210c9bdbddf3bc9519702cf927fa6767a40463b96f",
+      "admin_key": {
+        "_type": "ED25519",
+        "key": "1E9D38B5274152A78DFF1A86FA464CEADC1F4238CA2C17060C3C507349424A34"
```
Comment (mgoelswirlds) [inline docs/design/dynamic-addressbook.md:107]: This a 32 byte HEX string.

Comment (steven-sheehy) [reply docs/design/dynamic-addressbook.md:107]: Hex is always lowercase in our response. Not sure why we didn't just copy from an actual api response...

Comment (mgoelswirlds) [reply docs/design/dynamic-addressbook.md]: Didn't have a ED25519  response.Changed it.

**Review by steven-sheehy — APPROVED** [review]

**Review by jascks — APPROVED** [review]
LGTM

**Pull request merged**